packages/eslint-config-evg/config.js

```diff
@@ -0,0 +1,228 @@
+const ERROR = "error";
+const WARN = "warn";
+const OFF = "off";
+
+const errorIfStrict = process.env.STRICT ? ERROR : WARN;
+
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    jest: true,
+    node: true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:import/recommended",
+    "plugin:jsdoc/recommended-typescript-error",
+    // Airbnb includes some helpful rules for ESLint and React that aren't covered by recommended.
+    // See https://github.com/airbnb/javascript/tree/master/packages for specific rules.
+    "airbnb",
+    "prettier",
```

Comment:
prettier still does not work for me, and I wonder if it's because it should be this:
```suggestion
    "plugin:prettier/recommended",
```

I think we should fix it because it doesn't show up when I run `eslint:strict` and doesn't fix itself when I run `eslint:fix`. And because it doesn't fix itself for me when I save in VSCode 🥹

This is how it would look on main right now:
<img width="635" alt="Screenshot 2024-04-29 at 12 25 30 PM" src="https://github.com/evergreen-ci/ui/assets/47064971/6fec445b-4eda-4eb8-bfbf-1d981a6d2003">

Reply:
Thanks for flagging, I was trying `eslint-config-prettier` [as recommended](https://prettier.io/docs/en/integrating-with-linters.html) but turns out they're actually pretty different. I reintroduced the plugin so things should work again!

Reply:
interesting!! maybe we can take another look in the future 🤔 for now since we're just consolidating stuff this looks great to me!